pkg/config/config.go

```diff
@@ -247,6 +247,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("syslog_tls_verify", true)
 	config.BindEnvAndSetDefault("cmd_host", "localhost")
 	config.BindEnvAndSetDefault("cmd_port", 5001)
+	config.BindEnvAndSetDefault("agent_ipc_host", "localhost")
+	config.BindEnvAndSetDefault("agent_ipc_port", 0)
```

Comment:
Do we have tests for this somewhere?

Reply:
I added some tests for the logic of whether the new IPC server should be enabled